### PR TITLE
flask app

### PR DESCRIPTION
**Pull request by jcraggs**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+run:
+	python main.py
```
Comment (rdkr) [inline Makefile]: missing newline at end of file

File: README.md [hunk 2]
```diff
@@ -3,4 +3,32 @@
...
+
+## Application
+### main.py
+This starts flask and currently returns `hello, world`.
+
+### Makefile
+A Makefile is a collection of rules. Each rule is a recipe to do a specific thing. In this instance when you type `make run` the flask server runs.
+
+### requirements.txt
+Contains a list of items to be installed using pip install. Run "pip install -r requirements.txt" to install the required packages.
```
Comment (rdkr) [inline README.md]: we could have `install` as a make target

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+Jinja2==2.10.1
+keyring==10.6.0
+keyrings.alt==3.0
+MarkupSafe==1.1.1
+pycrypto==2.6.1
+pygobject==3.26.1
+pyxdg==0.25
+SecretStorage==2.3.1
+six==1.11.0
+Werkzeug==0.15.2
```
Comment (rdkr) [inline requirements.txt]: this is correct, but very verbose (i.e. it lists every resolved dependency of flask). we can just require flask and allow it to figure out the flask dependencies itself.

**Comment by rdkr** [comment]
nice overall. the python could use a bit of linting but this is addressed in another task so no worries.

**Review by GovindSuresh — APPROVED** [review]

**Review by rdkr — APPROVED** [review]

**Pull request merged**